examples/storage_proof.rs

```diff
@@ -0,0 +1,69 @@
+use axiom_eth::{keccak::KeccakChip, EthChip, storage::EthStorageChip, Field, providers::get_block_storage_input};
+use clap::Parser;
+use halo2_base::{AssignedValue, Context, gates::RangeInstructions};
+use halo2_scaffold::scaffold::{cmd::Cli, run_eth};
+use serde::{Deserialize, Serialize};
+use ethers_core::{types::{H160, H256}};
+use ethers_providers::{Provider, Http};
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct CircuitInput {
+    pub provider_url: String,
+    pub state_root: String,
+    pub block_number: u32,
+    pub addr: String,
+    pub slot: u64,
+    pub acct_pf_max_depth: usize,
+    pub storage_pf_max_depth: usize,
+}
+
+fn compute_storage_proof_at_slot<F: Field>(
```

Comment:
Specify a bit more clearly what this is constraining:

- If it is a proof that `slot` for `addr` at `block_number` has a given value, then this needs to be proven relative to a `block_hash`.
- A name like `verify_storage_proof` is more descriptive, since you aren't computing the proof in any sense.